setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup, find_packages
 
 setup(name='tap-looker',
-      version='0.0.1',
+      version='0.0.2',
```

Comment:
For future PR,  please leave this to us.

Reply:
Ok